app/components/NavigationBarWrapper.tsx

```diff
@@ -1,31 +1,21 @@
-import styled from '@emotion/native';
-import { useTheme } from 'emotion-theming';
-import PropTypes from 'prop-types';
-import * as React from 'react';
-import { Dimensions, StatusBar, Platform } from 'react-native';
+import React from 'react';
+import { SafeAreaView, TouchableOpacity, View, StyleSheet } from 'react-native';
 import { SvgXml } from 'react-native-svg';
 
-import { Icons } from '../assets';
-import { Colors } from '../styles';
+import { useStatusBarEffect } from '../navigation';
+import { Typography } from './Typography';
 
-/**
- * Navigation bar and status bar. Optionally include bottom nav
- *
- * @param {{
- *   title: string,
- *   onBackPress: () => void,
- *   includeBottomNav: boolean
- * }} param0
- */
-const widthScale = Math.min(Dimensions.get('window').width / 400, 1.0);
+import { Icons } from '../assets';
+import { Spacing, Colors, Typography as TypographyStyles } from '../styles';
 
 interface NavigationBarWrapperProps {
   children: React.ReactNode;
   title: string;
-  onBackPress: () => void;
+  onBackPress?: () => void;
```

Comment:
IMO this would be a good time to only have `onBackPress`, and remove `includeBackButton`, since this is now optional